charts/openmetadata/values.yaml

```diff
@@ -389,5 +389,11 @@ startupProbe:
     path: /healthcheck
     port: http-admin
 
+podDisruptionBudget:
+  enabled: true
```

Comment:
By default, it should be enabled.
```suggestion
  enabled: false
```

Reply:
Changing the code to have PDB disabled by default.